packages/demo/src/configuration/blocks/effects/spritesheetBlock.fragment.glsl

```diff
@@ -0,0 +1,23 @@
+uniform sampler2D input; // main
+uniform float time;
+uniform float rows;
+uniform float cols;
+uniform float frames;
+
+vec4 mainImage(vec2 vUV) { // main
+    float invRows = 1.0 / rows;
```

Comment:
Random, but I discovered that reciprocals are usually optimized in shaders (in a way that avoids the regular, expensive division operation)